Apps/W1/DynamicsGPHistoricalData/app/src/Pages/HistPayablesApply.Page.al

```diff
@@ -0,0 +1,180 @@
+                {
+                    ToolTip = 'Specifies the value of the Apply To Post Date field.';
+                }
+                field("Apply From Document No."; Rec."Apply From Document No.")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Document No. field.';
+                }
+                field("Apply From GL Posting Date"; Rec."Apply From GL Posting Date")
+                {
+                    ToolTip = 'Specifies the value of the Apply From GL Posting Date field.';
+                }
+                field("Apply From Currency Code"; Rec."Apply From Currency Code")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Currency ID field.';
+                }
+                field("Apply From Apply Amount"; Rec."Apply From Apply Amount")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Apply Amount field.';
+                }
+                field("Apply From Disc. Taken Amount"; Rec."Apply From Disc. Taken Amount")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Discount Taken Amount field.';
+                }
+                field("Apply From Disc. Avail. Taken"; Rec."Apply From Disc. Avail. Taken")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Discount Available Taken field.';
+                }
+                field("Apply From Write Off Amount"; Rec."Apply From Write Off Amount")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Write Off Amount field.';
+                }
+                field("Actual Apply To Amount"; Rec."Actual Apply To Amount")
+                {
+                    ToolTip = 'Specifies the value of the Actual Apply To Amount field.';
+                }
+                field("Actual Discount Taken Amount"; Rec."Actual Discount Taken Amount")
+                {
+                    ToolTip = 'Specifies the value of the Actual Discount Taken Amount field.';
+                }
+                field("Actual Disc. Available Taken"; Rec."Actual Disc. Available Taken")
+                {
+                    ToolTip = 'Specifies the value of the Actual Discount Available Taken field.';
+                }
+                field("Actual Write Off Amount"; Rec."Actual Write Off Amount")
+                {
+                    ToolTip = 'Specifies the value of the Actual Write Off Amount field.';
+                }
+                field("Apply From Exchange Rate"; Rec."Apply From Exchange Rate")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Exchange Rate field.';
+                }
+                field("Apply From Denom. Exch. Rate"; Rec."Apply From Denom. Exch. Rate")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Denomination Exchange Rate field.';
```

Comment:
wrong field value in tooltip? Denomination instead of Denom. Other fields use the same abbreviation as the field.